### PR TITLE
Added styles and render logic to reduce load jank

### LINKED ISSUE #38: page loading at the bottom
### Describe your question

hi thanks so much for this plugin! I'm coming across an issue of my page loading at the bottom (where the player is located) but I need the page to load at the top. Do you know what I could do to fix this?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by iansvo**
### Description of the Change

Closes [#38](https://github.com/10up/retro-winamp-block/issues/38)

This adds some CSS to remove the element from the document flow (via absolute positioning) and using a timer that starts when the load event fires in the browser to then remove the absolute positioning. 

The effect is that the element is inserted into the DOM after its dimensions are calculated, which prevents the scrolling.

### Possible Drawbacks

The timeout value is at 1000ms, which is kind of arbitrary. There isn't a complete way to guarantee perfect loading as the widget could delay for any number of reasons unrelated to the webpage (such as network speed, device resources, and so on).

It is possible we might want to hook into this using some other mechanism (mutation change, perhaps?) but that rabbit hole can get deep really quickly and I think the solve should start simpler and move from there after we do some real world testing. 

### Verification Process

I started with replicating the issue locally on a barebones page that didn't have much on it besides the widget. I was able to replicate the issue with relative consistency when the page was not cached.

After the update I went through the same steps and found the issue was not able to be reproduced. 

**Note**: This issue should be verified on a staging/production environment using a configured theme and a more realistic page setup to confirm the durability of the solution. Hopefully this is something triage in the related issue can resolve. 

### Checklist:

- [X] I have read the [**CONTRIBUTING**](https://github.com/10up/.github/blob/trunk/CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation. (I think this is FAQ material at least)
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change. (N/A)
- [X] All new and existing tests passed.

### Changelog Entry

Added - Additional handling to prevent page jank and scrolling as the webamp widget loads

### Credits

Props @iansvo

### CODE REVIEW

**Comment by iansvo** [comment]
@jeffpaul I just noticed that the JS lint fails, it's saying there's a package missing which my changes didn't alter either. Possible version issue with the deps?

File: src/frontend.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Webamp from 'webamp';
-import domReady from '@wordpress/dom-ready';
```
Comment (cadic) [inline src/frontend.js:2]: `@wordpress/dom-ready` is no more a dependency for this and other scripts, could be removed from `package.json`

**Review by cadic — DISMISSED** [review]
The code looks good, just a potential cleanup for node packages

File: src/frontend.js (hunk 1 above)
Comment (jeffpaul) [reply src/frontend.js:2]: @iansvo if you concur with ^, mind a quick commit to pull that out?

Comment (jeffpaul) [reply src/frontend.js:2]: @cadic could you handle that update and then look to get this merged in?

Comment (iansvo) [reply src/frontend.js:2]: @cadic @jeffpaul Update is made confirmed build succeeds, merged in latest from dev and regenerated package-lock.json.

**Comment by jeffpaul** [comment]
@iansvo mind helping resolve the merge conflicts and then @cadic can do final review/merge?

**Comment by jeffpaul** [comment]
@10up/open-source-practice would be great to get some assistance resolving the merge conflicts here so that this can get into final review/merge... thanks!

**Comment by peterwilsoncc** [comment]
@10up/open-source-practice I've resolved the merge conflicts but am unable to reproduce the issue described so will need someone else to test and review.

**Review by Sidsector9 — APPROVED** [review]
Thanks for resolving the conflicts @peterwilsoncc 

I was not able to reproduce this issue locally however I was able to on a live site.
I used [instawp](https://instawp.com/) to test.

### Issue on the develop branch:

https://user-images.githubusercontent.com/17757960/213983598-5a862861-efa9-4b27-b58f-357c973e3113.mov

### Issue resolved in the fix brach:


https://user-images.githubusercontent.com/17757960/213983661-1cf12939-1b7e-4557-82f2-b766d83f184d.mov

I have tested on pages with and without content and it worked well for me.

**Pull request merged**